roles/kubernetes-apps/scheduler_plugins/tasks/patch_kube_scheduler.yml

```diff
@@ -0,0 +1,65 @@
+---
+- name: Scheduler Plugins | Backup kube-scheduler.yaml
+  copy:
+    src: "{{ kube_manifest_dir }}/kube-scheduler.yaml"
+    dest: "{{ kube_config_dir }}/kube-scheduler.yaml.bak"
+    remote_src: true
+    mode: preserve
+    force: false     # backup only once
+  when: inventory_hostname in groups['kube_control_plane']
+  tags:
+    - scheduler_plugins
+
+- name: Scheduler Plugins | Create yq patch for kube-scheduler
+  template:
+    src: kube-scheduler-patch.yq.j2
+    dest: "{{ kube_config_dir }}/scheduler-plugins/kube-scheduler-patch.yq"
+    mode: 0644
+  register: kube_scheduler_yq_patch
+  when: inventory_hostname in groups['kube_control_plane']
+  tags:
+    - scheduler_plugins
+
+- name: Scheduler Plugins | Download yq
+  include_tasks: "../../../download/tasks/download_file.yml"
+  vars:
+    download: "{{ download_defaults | combine(downloads.yq) }}"
+  when: inventory_hostname in groups['kube_control_plane']
+  tags:
+    - scheduler_plugins
+
+- name: Scheduler Plugins | Copy yq binary from download dir
+  copy:
+    src: "{{ downloads.yq.dest }}"
+    dest: "{{ bin_dir }}/yq"
+    mode: 0755
+    remote_src: yes
+  when: inventory_hostname in groups['kube_control_plane']
+  tags:
+    - scheduler_plugins
```

Comment:
We should not include download role tasks like that... (I know it's done all over the place, but it's really not a good idea).
The download role can download it iself, can't it ?